back/src/main/java/com/baba/back/content/controller/ContentController.java

```diff
@@ -30,18 +30,19 @@ public class ContentController {
 
     private final ContentService contentService;
 
-    @Operation(summary = "컨텐츠 생성 요청")
+    @Operation(summary = "성장 앨범 생성 요청")
     @CreatedResponse
     @BadRequestResponse
     @UnAuthorizedResponse
     @ForbiddenResponse
     @NotFoundResponse
     @IntervalServerErrorResponse
     @PostMapping(value = "/album/{babyId}", consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
-    public ResponseEntity<CreateContentResponse> createContent(@ModelAttribute @Valid CreateContentRequest request,
-                                                               @Login String memberId,
-                                                               @PathVariable("babyId") String babyId) {
-        return ResponseEntity.status(HttpStatus.CREATED).body(contentService.createContent(request, memberId, babyId));
+    public ResponseEntity<Void> createContent(@ModelAttribute @Valid CreateContentRequest request,
+                                              @Login String memberId,
+                                              @PathVariable("babyId") String babyId) {
+        final Long contentId = contentService.createContent(request, memberId, babyId);
+        return ResponseEntity.created(URI.create("/api/album/" + babyId + "/" + contentId)).build();
```

Comment:
```suggestion
        return ResponseEntity.created(URI.create("/album/" + babyId + "/" + contentId)).build();
```
클라이언트 쪽에서 `dev.baba.kids/api`가 base url인 것을 알고 있으니까 url에 api를 빼는게 어떨까요?

Reply:
제거했습니다!\